test/t8n/t8n.cpp

```diff
@@ -137,6 +138,7 @@ int main(int argc, const char* argv[])
 
                         j_receipt["transactionHash"] = hex0x(computed_tx_hash);
                         j_receipt["gasUsed"] = hex0x(static_cast<uint64_t>(receipt.gas_used));
+                        blockGasUsed += receipt.gas_used;
```

Comment:
This is also `"cumulativeGasUsed"` so I propose to name the variable like this and assign correct value to the receipt. The block gas used it the value of the last receipt.